docs/aries-cloud-agent-demo.md

```diff
@@ -90,17 +99,35 @@ At this point, the two ACAs are connected to each other.
 
 ### Dependencies
 
-Follow the <a href="../quickstart/#preliminaries">Preliminaries</a> and <a href="../quickstart/#installation">Installation</a> sections from the AEA quick start.
+Follow the <a href="../quickstart/#preliminaries">Preliminaries</a> and <a href="../quickstart/#installation">Installation</a> sections from the AEA quick start. 
 
 Install Aries cloud-agents (for more info see <a href="https://github.com/hyperledger/aries-cloudagent-python#install" target=_blank>here</a>) if you do not have it on your machine:
 
 ``` bash
 pip install aries-cloudagent
 ```
 
+This demo has been successfully tested with aca-py version 0.4.5.
+
+This demo requires an instance of von network running in docker locally (for more info see <a href="https://github.com/bcgov/von-network#running-the-network-locally" target=_blank>here</a>)
+
+This demo has been successfully tested with the von-network git repository pulled on 07 Aug 2020. 
+
 ### Terminals
 
-Open four terminals. Each terminal will be used to run one of the four agents in this demo.
+Open five terminals. The first terminal is used to run an instance of von-network locally in docker. The other four terminals will be used to run each of the four agents in this demo.
+
+## VON Network
+
+In the first terminal move to the `von-network` directory and run an instance of `von-network` locally in docker.
+
+This <a href="https://github.com/bcgov/von-network#running-the-network-locally" target=_blank>tutorial</a> has information on starting (and stopping) the network locally.
+
+``` bash
+./manage build
+./manage start --logs
+``` 
+Once the ledger is running, you can see the ledger by going to the web server running on port 9000. On localhost, that means going to <a href="http://localhost:9000" target=_blank>http://localhost:9000</a>.  
```

Comment:
Please make clear that this is an optional part of the demo

Reply:
Without the ledger it wouldn't really be an identity demo. The AEAs connect with each other and connect with their respective ACAs and then do some basic communication framework side to find each other and send some data around. They wouldn't be able to achieve any identity interaction, unique to this demo. 

Even making ACAs introduced and connected to one another (in the Aries sense) requires Faber to have registered a DID on the ledger so Alice can pick that up.